pict-doc/pict/scribblings/pict.scrbl

```diff
@@ -543,7 +543,7 @@ argument for consistency with the other functions.}
                       [#:style style (one-of/c 'transparent 'solid 'xor 'hilite 
                                                'dot 'long-dash 'short-dash 'dot-dash 
                                                'xor-dot 'xor-long-dash 'xor-short-dash 
-                                               'xor-dot-dash)]
+                                               'xor-dot-dash) 'solid]
```

Comment:
It looks like implementation of `pin-arrows-line` passes `#f` as the default style argument, although the docs don't say false is allowed?

https://github.com/racket/pict/blob/master/pict-lib/pict/private/main.rkt#L145